preview/launcher/powershell-wrapper

```diff
@@ -3,6 +3,6 @@
 # Licensed under the MIT License.
 
 # Create $XDG_RUNTIME_DIR if it doesn't exist
-[ -n "$XDG_RUNTIME_DIR" ] && mkdir -p $XDG_RUNTIME_DIR -m 700
+[ -n "$XDG_RUNTIME_DIR" ] && mkdir -p $XDG_RUNTIME_DIR -m 700 || true
```

Comment:
Why is this needed?

Reply:
some environments (Azure devops) have incorrect permissions on this folder and it cannot be created

Reply:
We should maybe add a comment about that

Reply:
```suggestion
# || true causes bash to ignore errors
[ -n "$XDG_RUNTIME_DIR" ] && mkdir -p $XDG_RUNTIME_DIR -m 700 || true
```